web/libs/editor/public/styles/main.css

```diff
@@ -2,6 +2,49 @@ body {
   position: relative;
   margin: 0;
   padding: 0;
+
+  --sand_900: #12110D;
```

Comment:
these colors are defined in web/apps/labelstudio/src/themes/default/colors.scss - are comments not being pulled into a spot that pulls from that?

Reply:
`web/apps/labelstudio` is LSO, but we have LSF standalone and it's broken a bit. There is also some problem with `border` in some elements.

Reply:
For example:
<img width="332" alt="image" src="https://github.com/user-attachments/assets/e40224d0-f63c-4ab5-b236-36fd0ebdbfa9">
<img width="415" alt="image" src="https://github.com/user-attachments/assets/c1628f8c-abe7-4122-9ea2-53606227b950">
instead of
<img width="343" alt="image" src="https://github.com/user-attachments/assets/c139f56c-6bb9-4a02-9ee6-12b4de53334e">
<img width="328" alt="image" src="https://github.com/user-attachments/assets/c242fdf3-7400-4b47-8e73-ffc061a18ff7">